src/common.apib

```diff
@@ -44,12 +44,12 @@
 
 ## Publisher Database Base (object)
 + database_name: PHG Test Database (required)
-+ creation_method: Email
++ creation_method: Email (required)
```

Comment:
Is it worth splitting these attributes out, into an optional update object, and overriding the required in the base?